JavaAssignments/src/ankush/SquareCube.java

```diff
@@ -0,0 +1,22 @@
+package ankush;
+
+public class SquareCube {
+	
+	//Method to Calculate Square of a Number
+	static void square(int number)
```

Comment:
It is always advisable to return value from method whenever possible. Here caller may need answer for further processing.